frontend/src/components/table/useCheckboxTable.ts

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { xor } from 'lodash';
-import Table from '~/components/table/Table';
+import { Table } from '~/components/table';
```

Comment:
Careful with this -- you should import directly from the file in question and not use the index import for those items inside the index import. Circle references can occur in JavaScript Modules. I think there is decent protection these days and TypeScript doesn't suffer from this but it seems weird on face value to "have an index that is also imported by those inside the index"

Reply:
An alternative would be to import just the type -- I don't see a need to import the component itself (ie, only do it for the TS compiler) -- tree shaking might prevent this in the webpack bundle 🤔 

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export

Reply:
Good call, tried to use `import type Table from './Table'` and it works pretty good.